File: stringE /stringE.swift
import UIKit

func stringE(str:String, char:Character) ->Bool {
    var letters = Array(str)
    var count = 0
    for letter in letters {
        if letter == char {
            count += 1
        }
    }
    if(count>=1 && count<=3){
        return true
    }
    return false
}
print(stringE(str: "Heelloee", char: "e"))

File: stringE /stringE.cs
namespace Testing_Exercises
{
    class MainClass
    {
        public static void Main(string[] args)
        {
            Console.WriteLine(stringE("Helloeee"));
        }
        public static bool stringE(String str)
        {
            int count = 0;
            for (int i = 0; i < str.Length; i++)
            {
                if (str[i] == 'e')
                {
                    count++;
                }
            }

            return 1 <= count && count <= 3;
        }
    }
}
